controllers/fcToolsController.js

```diff
@@ -17,73 +17,151 @@ exports.fitTool = function(req, res) {
-                        users.checkSkills(targetUser, skillsPackage, function(skills) {
-                            res.render('toolsSkills.njk', {userProfile, sideBarSelected, skills, targetUser});
-                        })
-                    } else {
-                        log.error("fcTools.skillsChecker: Error for users.findAndReturnUser: ", { name: req.params.pilotname });
-                        req.flash("content", {"class":"error", "title":"Woops!", "message":"We couldn't find " + req.params.pilotname + ". Have they used our system before?"});
-                        res.redirect(`/commander`);                        
-                    }
+/*
+* Returns the pilot information page
+* @params req{}
+* @return Renders view
+*/
+exports.pilotSearch = function(req, res){
+    if(!req.isAuthenticated() || req.isAuthenticated() && req.user.role.numeric < 1){
+        req.flash("content", {"class":"error", "title":"Not Authorised!", "message":"Only our FC team has access to that page! Think this is an error? Contact a member of leadership."});
+        res.status(403).redirect("/");
+        return;
+    }
+
+    esi.characters.search.strict(req.params.pilotname.replace(/-/g,' ')).then(function (results) { 
+        users.findAndReturnUser(Number(results), function(targetUser){
+            if(targetUser) {
+                users.getAlts(targetUser.characterID, function(alts){
+                    var userProfile = req.user;
+                    targetUser.account.pilots = alts;
+                    var sideBarSelected = 6;
+                    res.render('toolsPilotSearch.njk', {userProfile, sideBarSelected, targetUser});
                 })
-            }).catch(function (err) {
-                log.error("fcTools.skillsChecker: Error for esi.characters.search", { err, name: req.params.pilotname });
-                req.flash("content", {"class":"error", "title":"Woops!", "message":"We couldn't find " + req.params.pilotname + ". Did you spell your pilot name correctly?"});
-                res.redirect(`/commander`);
+            } else {
+                req.flash("content", {"class":"error", "title":"Woops!", "message":"We couldn't find " + req.params.pilotname + ". Have they used our system before?"});
+                res.redirect(`/commander`);                        
+            }
+        })
+    }).catch(function (err) {
+        req.flash("content", {"class":"error", "title":"Woops!", "message":"We couldn't find " + req.params.pilotname + ". Did you spell your pilot name correctly?"});
+        res.redirect(`/commander`);
+    })
+}
+
+/*
+* AJAX Search for the pilot
+* @params req{}
+* @return res{} url for pilot page || error
+*/
+exports.searchForPilot = function(req, res){
+    if (req.isAuthenticated() && req.user.role.numeric > 0) {
+        esi.characters.search.strict(req.body.search).then(function (results) {
+            //ESI cannot find pilot
+            if(results == undefined){
```

Comment:
Lets use `!!` here. That handles all cases of `undefined` and `null`

Reply:
Done 
```
esi.characters.search.strict(req.body.search).then(function (results) {
    //ESI cannot find pilot
    if(!!results){
```